Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:apache
+MAINTAINER David A. Lareo "dalareo@gmail.com"
+
+RUN apt-get update -y \
+	&& apt-get install -y git libmcrypt-dev libldap2-dev \
+	&& docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+
+	&& docker-php-ext-install mcrypt ldap
+
+RUN git clone https://github.com/simplesamlphp/simplesamlphp.git /var/simplesamlphp
```

Comment:
I guess the problem here is that you are thinking about using this to deploy something easily into production, while I'm thinking about this to make it easier for developers... 😄 

Maybe having a separate `Dockerfile` for development?